Board.py

```diff
@@ -1,11 +1,163 @@
-from checker import Checker
+from enum import Enum
+
+ROWS = 8
+COLS = 8
+
+
+class Player(Enum):
+    player1 = 0
+    player2 = 1
+
+
+class Piece(Enum):
+    basic = 0
+    king = 0
+
+
+def in_bounds(x, y):
+    return 0 <= x < COLS and 0 <= y < ROWS
+
+
+def is_jump(start_x, end_x):
+    return abs(end_x - start_x) > 1
 
 
 class Board:
+
     def __init__(self):
-        rows, cols = 8, 8
-        self.board = [['0' for _ in range(cols)] for _ in range(rows)]
+        self.player_one_pieces = []
+        self.player_two_pieces = []
+        for i in range(3):
```

Comment:
where does the number 3 come from? if this is always fixed to 3 no matter what, make this a magic value (see line 3)

Reply:
nit: more descriptive name than i